docs/articles/documentation/guides/basic-guides/select-page-elements.md

```diff
@@ -157,6 +157,33 @@ const sel = Selector('div').child();
 </html>
 ```
 
+The expression, associated with a selector is evaluated when this selector is used, not when it is declared. If page's content changes, subsequent selectors are reevaluated reactively.
```

Comment:
> reactively

Andrey wrote 'reactively' but I don't really understand why it's needed and what it means